examples/package.json

```diff
@@ -29,14 +29,15 @@
     "start": "start-storybook -p 6006 -s ./public --ci ",
     "storybook:build": "build-storybook -s ./public",
     "storybook:deploy": "npm run storybook:build && gh-pages -d storybook-static",
+    "storybook:just_deploy": "gh-pages -d storybook-static",
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
     "@deck.gl/core": "6.3.3",
     "@deck.gl/layers": "6.3.3",
     "@deck.gl/react": "6.3.3",
-    "@flowmap.gl/core": "^5.2.0",
-    "@flowmap.gl/react": "^5.2.0",
+    "@flowmap.gl/core": "file:../packages/core",
```

Comment:
How do you setup your dev workflow? 
I had to use relative dependencies and three hot loading compilers in order to get a flow that allowed changes from core to be propagated to the story book

Reply:
You should run `npm run core-dev` in parallel which would recompile the core automatically and the changes should be picked up.

Reply:
i was doing that... but it seemed like the examples were just getting the 5.2.0 version that was installed into the node_modules directory?
I'm not very experienced with the npm build system so I could be misunderstanding though.

Reply:
When you run `npm run bootstrap` it creates symlinks to the subpackages within `node_modules` so that you can work with the versions of the packages you have in your local copy of the repo.